app/controllers/api/v1/search_terms_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
+    class SearchTermsController < ApplicationController
+      def import
+        return render :json => { :errors => 'No file added' }, :status => 422 if params[:file].nil?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/ffe479ef97670c7333f37452d55369f226bbc764/app/controllers/api/v1/search_terms_controller.rb#L5"></span>Prefer <code>:unprocessable_entity</code> over <code>422</code> to define HTTP status code.</td>
    </tr>
  </tbody>
</table>